brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/Alien4CloudEntitySpecFactory.java

```diff
@@ -47,10 +52,11 @@ public Alien4CloudEntitySpecFactory(ManagementContext mgmt, ToscaFacade<Alien4Cl
             spec = (EntitySpec<?>) mgmt.getCatalog().createSpec(catalogItem);
 
         } else if (isComputeType(nodeId, toscaApplication)) {
-            spec = hasMultipleChildren
-                   ? EntitySpec.create(SameServerEntity.class)
-                   : EntitySpec.create(BasicApplication.class);
-
+            spec = EntitySpec.create(SameServerEntity.class);
+            spec.enricher(Enrichers.builder().transforming(Attributes.ADDRESS)
```

Comment:
Well it is an attribute that is only defined on Compute in the [Alien normative types](https://github.com/alien4cloud/tosca-normative-types/blob/master/normative-types.yml#L393), but it is incorrectly defined - according to the spec that attribute shouldn't be on Compute - but it is required to get the Mongo example working!